messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/bridge/JMSBridgeDefinition.java

```diff
@@ -200,6 +200,11 @@
             .setFlags(AttributeAccess.Flag.STORAGE_RUNTIME)
             .build();
 
+     public static final SimpleAttributeDefinition ABORTED_MESSAGE_COUNT = create("aborted-message-count", LONG)
+            .setStorageRuntime()
+            .setUndefinedMetricValue(new ModelNode(0L))
```

Comment:
this `undefined metric value` should be set only if the metric is not always enabled (and there should then be a corresponding  `statistics-enabled` attribute.
The metrics are read from Artemis' `JMSBridge`. Are they always returning a meaningful value?

Reply:
JMS Bridge statistics are always enabled.

Reply:
Brian is right: to comply with https://issues.jboss.org/browse/WFCORE-831, when the server is in admin-only mode, the `:read-attribute` operation for the metrics must succeed and returns  `undefined`.